sdk/php/src/Dagger.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Dagger;
+
+use CompileError;
+use Dagger\Dagger\DaggerClient;
```

Comment:
We really want to merge & release this, but I also think that we should reduce some of this nesting. `Dagger\Dagger\DaggerClient` is very repetitive and I don't see a reason why we would need this. `Dagger\Client` is what I would expect this to be.

Most of the generated code has `Dagger\Dagger` which I think it should just be `Dagger`.